programs/voter-stake-registry/src/instructions/withdraw.rs

```diff
@@ -87,12 +87,18 @@ pub fn withdraw(ctx: Context<Withdraw>, deposit_entry_index: u8, amount: u64) ->
     let registrar = &ctx.accounts.registrar.load()?;
     let voter = &mut ctx.accounts.voter.load_mut()?;
 
+    // Get the exchange rate for the token being withdrawn.
+    let mint_idx = registrar.voting_mint_config_index(ctx.accounts.destination.mint)?;
+
     // Governance may forbid withdraws, for example when engaged in a vote.
-    let token_owner_record = voter.load_token_owner_record(
-        &ctx.accounts.token_owner_record.to_account_info(),
-        registrar,
-    )?;
-    token_owner_record.assert_can_withdraw_governing_tokens()?;
+    // Not applicable for tokens that don't contribute to voting power.
```

Comment:
nitpick: do you want to give an example? the only one I can think of is - grant in usdc (without voting power) with vesting schedule which can be clawed back